recipe/meta.yaml

```diff
@@ -38,7 +37,7 @@ requirements:
     - libgfortran  # [osx]
     - blas 1.* {{blas}}
     {% if blas == 'openblas' %}
-    - openblas 0.2.20|0.2.20.*
+    - openblas 0.2.20
```

Comment:
Why is the openblas pin updated here?

Reply:
Sorry, I didn't reply here but in the PR. So, what should I do here? Do you want me to revert this change?

Reply:
I think reverting this is the right thing to do. Only change it if it's fixing something.